sql/core/src/test/resources/sql-tests/inputs/csv-functions.sql

```diff
@@ -0,0 +1,12 @@
+-- from_csv
+describe function from_csv;
+describe function extended from_csv;
```

Comment:
Shall we not test this? It requires to fix the tests accordingly everytime we fix the doc.

Reply:
Do you mean both `describe` or only the last one? I would actually leave it to keep consistency with `from_json` tests.

Reply:
I moved most of the tests for `csvFunctionsSuite` to `csvExpressionsSuite`. I leaved 2 tests for testing both signatures of the `from_csv` method.